### DISCUSSION TITLE
Include all BTTV and 7TV emotes enabled by streamer in emote menu

### DISCUSSION THREAD

**Discussion by exatedlord** [night/betterttv#6923] [Ideas]
In the emote menu next to the chat input box, you can see all Global, channel and frequently used emotes. However, in 7TV, you can see every single BTTV and 7TV emote enabled for the channel. 

I included comparison pictures:
![7tv emote menu](https://github.com/night/betterttv/assets/52119093/0731464d-5088-49e0-bab0-43013deddcfa)
![BTTV emote menu](https://github.com/night/betterttv/assets/52119093/a86096d3-555f-486d-9f39-03d8fc0a27b0)

**Comment by night** [comment]
BetterTTV has an emote menu that is, in our opinion, much better than the others, and includes our emotes. You can turn it on in the BetterTTV settings.
